stremio-core-web/src/model/serialize_ctx.rs

```diff
@@ -17,17 +17,17 @@ mod model {
 
     use stremio_core::deep_links::SearchHistoryItemDeepLinks;
     use stremio_core::types::{
-        events::Events, notifications::NotificationItem, profile::Profile, resource::MetaItemId,
+        events::Events, notifications::NotificationItem, resource::MetaItemId,
     };
     use url::Url;
 
+    use crate::env::WebEnv;
     use crate::model::deep_links_ext::DeepLinksExt;
```

Comment:
This is preferred in core nowadays:
```suggestion
    use crate::{env::WebEnv, model::deep_links_ext::DeepLinksExt};
```